Engine/debug/debug.cpp

```diff
@@ -638,10 +636,10 @@ void scriptDebugHook (ccInstance *ccinst, int linenum) {
 
     const char *scriptName = ccinst->runningInst->instanceof->GetSectionName(ccinst->pc);
 
-    for (size_t i = 0; i < breakpoints.size(); ++i)
+    for (auto & breakpoint : breakpoints)
```

Comment:
This also should be `const auto &breakpoint`.

Reply:
added! :)